apps/smart-contracts/core/test/integration/DepositTradeHelper.test.ts

```diff
@@ -0,0 +1,104 @@
+import chai from 'chai'
+import { ethers, network } from 'hardhat'
+import { smock } from '@defi-wonderland/smock'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address'
+import { BigNumber } from 'ethers'
+import { getPrePOAddressForNetwork } from 'prepo-constants'
+import { utils } from 'prepo-hardhat'
+import { parseEther } from '@ethersproject/units'
+import { create2DeployerFixture } from '../fixtures/Create2DeployerFixtures'
+import { Snapshotter } from '../snapshots'
+import { MockCore } from '../../harnesses/mock'
+import { roleAssigners } from '../../helpers/roles'
+import { PrePOMarketParams } from '../../types'
+import { Create2Deployer, ERC20 } from '../../types/generated'
+import { ERC20AttachFixture } from '../fixtures/ERC20Fixture'
+
+const { nowPlusMonths } = utils
+
+chai.use(smock.matchers)
+const snapshotter = new Snapshotter()
+
+describe('=> DepositTradeHelper', () => {
+  let weth: ERC20
+  let core: MockCore
+  let deployer: SignerWithAddress
+  let governance: SignerWithAddress
+  let defaultMarketParams: PrePOMarketParams
+  let create2Deployer: Create2Deployer
+  const TEST_NAME_SUFFIX = 'preSPACEX 2000-10000 31-December 2022'
+  const TEST_SYMBOL_SUFFIX = 'preSPACEX_2000_10000_31_DEC_22'
```

Comment:
nit: new naming format
```suggestion
  const TEST_NAME_SUFFIX = 'Fake Token ($5-10B)'
  const TEST_SYMBOL_SUFFIX = 'FAKE_5-10B'
```